generators/app/templates/javascript/_package.json

```diff
@@ -32,7 +36,7 @@
     "gulp-nodemon":"^2.0.7",
```

Comment:
if nodemon is the only thing we use gulp for, then let's just go ahead with plain nodemon, no need for gulp